api/v1alpha1/databaseengine_types.go

```diff
@@ -51,16 +52,11 @@ const (
 )
 
 const (
-	// DatabaseOperatorUpgradeAnnotation indicates that the database operator needs to be upgraded.
-	// The value of the annotation is the version to upgrade to.
-	DatabaseOperatorUpgradeAnnotation = "everest.percona.com/upgrade-operator-to"
 	// DatabaseOperatorUpgradeLockAnnotation is an annotation set on the database engine.
-	// If present and set to "true", Everest backend will prevent any resources from being modified in the namespace.
-	// This annotation is typically set by the Everest backend right before starting the operator upgrade.
+	// If present, the value should contain the timestamp of when the lock was set.
+	// Everest operator automatically removes this annotation (lock) after 5mins.
```

Comment:
Are you sure 5 minutes is actually enough? What if the cluster is very slow at downloading images?

Reply:
5mins was assuming most people run it on AWS and GKE, where I think 5mins would have been sufficient to download everything.. But I'm fine with a higher value too

Reply:
I'm not sure what's the most appropriate value to be honest :confused:

Reply:
Okay, let's change it to 10mins, to be on the safer side :)